### ISSUE TITLE
Errors from graphql-tag are not reported

### ISSUE THREAD

**Issue by yurtsiv** [Swydo/meteor-graphql#51]
Hi, 

when a `.graphql` file has syntax errors, Meteor just reports  `Error: Cannot find module 'myFile.graphql'`
instead of printing an error from `graphql-tag` loader.

f.e.

/schema.graphql
```graphql
# 'typ' instead of 'type'
typ Query
   test: String
}
```

/index.ts
```javascript
import schema from './schema.graphql';
```

will fail with `Cannot find module './schema.graphql'`. When I fix the syntax error, everything works fine.

I've played around with the package a  bit, and the `file.error` get's called  [here](https://github.com/Swydo/meteor-graphql/blob/master/compiler.js#L26), but Meteor seems to ignore that for some reason.

I'm not sure if it's a problem with the package or Meteor or my code.

I use `METEOR@1.11.1`

Thanks!

**Comment by kolyasya** [comment]
@yurtsiv yeah, thank you for reporting this, I spent 10 minutes trying to understand why Meteor doesn't see my .graphql file to import.
Ended up just with an issue in the file itself, imports fine after fix
